eqllib/analytics/collection/T1005-txt-files-with-command-line-redirection.toml

```diff
@@ -0,0 +1,23 @@
+[analytic.metadata]
+categories = ["detect"]
+confidence = "medium"
+contributors = ["Daniel Stepanic"]
+created_date = "12/04/2019"
+description = "Identifies Koadic's use of redirecting command output to plaintext files."
+id = "7c7f3114-7bdd-4477-a4e0-b5105b6babd8"
+name = "Creation of TXT File via cmd.exe with Command Line Redirection"
```

Comment:
I believe we did this for Koadic:
https://eqllib.readthedocs.io/en/latest/analytics/7c7f3114-7bdd-4477-a4e0-b5105b6babd8.html

If it's different, we should update the existing one